README.md

```diff
@@ -305,34 +305,36 @@ Une fois cloné ce répertoire à l'aide de
 
     git clone git@github.com:etalab/sirene_as_api.git && cd sirene_as_api
 
-Construisez le container avec `docker-compose build` et lancez le avec `docker-compose up`.
+Construisez le container avec `docker-compose build` et lancez-le avec `docker-compose up`.
+
+Pour faciliter l'intéraction avec le container du serveur, il est conseillé d'utiliser le mode intéractif (à partir d'un autre terminal) :
+
+    docker exec -it <nom_container> bash
+
+Vous pourrez ainsi exécuter, de manière transparente, toutes les tâches d'administration et de mises à jour présentées dans les sections suivantes. Sinon vous pouvez prendre exemple sur les commandes ci-dessous dans cette même section. 
 
 Vous pouvez effectuer les migrations (nécessaire seulement au premier lancement) à l'aide de :
 
-    docker-compose run sirene db:create
-    docker-compose run sirene db:migrate
+    docker-compose run sirene bundle exec rails db:create
+    docker-compose run sirene bundle exec rails db:migrate
 
-La base de donnée sera persistée sur `/var/lib/postgresql/data`.
+La base de donnée sera persistée dans le dossier `/var/lib/postgresql` par défaut. Il est possible  de changer l'emplacement d'installation des données ou d'indiquer un emplacement d'installation existante en modifiant la variable d'environnement `POSTGRES_DATA` dans le fichier `.env`.
+
+Si votre machine comprend déjà une base installée sur `/var/lib/postgresql`, libre à vous de modifier le fichier `.env` :
+```yml
+POSTGRES_DATA=/path/to/other/data_folder
+```
 
 Lancez les imports à l'aide des commandes rake (Cf plus bas, partie Mises à jour / Administration) précédées par `docker-compose run sirene`. Par exemple :
 
-    docker-compose run sirene rake sirene_as_api:import_last_monthly_stock
+    docker-compose run sirene rake sirene_as_api:populate_database
 
 Vous pouvez d'ors et déjà interroger l'API sur `http://localhost:3000/v1/siren/552032534`.
 
 La recherche fulltext devrait fonctionner après avoir executé `docker-compose run sirene rake sunspot:solr:reindex`. Les index solr sont persistés egalement.
 
 Attention : pour faciliter les modifications par nos utilisateurs, le docker est lancé en environnement de développement. N'oubliez pas de changer le mot de passe postgres dans /config/docker/init.sql, /config/docker/database.yml et docker-compose.yml.
 
-Si votre machine comprend déjà une base installée sur /var/lib/postgresql avec une version différente de postgres, vous aurez peut-être besoin de modifier les lignes suivantes du fichier dockerfile-compose.yml :
-
-```yml
-environment:
-  - PGDATA=/var/lib/postgresql_docker/data
-
-volumes :
-  - /var/lib/postgresql/data:/var/lib/postgresql_docker/data
```

Comment:
Alors la raison ici pour laquelle j'ai voulu séparer la database sirene_as_api_production / development de sirene_as_api_docker tiens aux permissions (j'ai eu des soucis à créer la bonne database avec le bon user sous Docker).

Du coup plusieurs éventualités :
- Avez vous testé cette situation ? (database "classique" et celle docker). Cela fonctionne-t-il ?
Si oui : je vais regarder encore une fois voir comment vous avez réussi ;)
Si non : il faudra au minimum documenter
- Vous n'avez pas testé :
Dans ce cas je tenterais soit de faire fonctionner, soit je documenterais la situation

Reply:
/var/lib/postgresql_docker/data est juste le dossier utilisé à l'intérieur du container postgres pour trouver ses données. Juste changer cette variable ne devrait pas avoir d'influence sur le comportement décrit. Le volume est toujours monté sur /var/lib/postgresql/data sur le host.

Je reformule la question : vous voulez utiliser les mêmes fichiers pour deux bases de données différentes, une "classique" non dockerized et une dockerized ?
Si c'est le cas, il y a effectivement risque de conflit de droits. Supposons que les fichiers d'une base sont créés par le container postgres et monté sur /var/lib/postgresql/data, ces fichiers sont owned par root donc si vous décidez d'utiliser votre base "classique" avec les fichiers de /var/lib/postgresql/data, vous risquez de ne pas avoir les droits nécessaires. Je ne pense pas que cela soit quelque chose qu'on veut faire de manière générale.

Ce post parle de ce problème si j'ai bien compris ?
https://stackoverflow.com/questions/35679995/how-to-use-a-postgresql-container-with-existing-data

Pour que ce soit + configurable, il faudrait qu'on puisse se connecter à la base de données qu'on veut

Reply:
Effectivement c'est bien ça. Mais comme vous le dites ce n'est probablement pas quelque chose qu'on veut faire... Je vais au minimum documenter cela.